### ISSUE TITLE
🐛 BUG:  create cloudflare does not use default name

### ISSUE THREAD

**Issue by punkeel** [cloudflare/workers-sdk#3530]
### Which Cloudflare product(s) does this pertain to?

C3

### What version of `Wrangler` are you using?

2.0.10, latest

### What operating system are you using?

Mac

### Describe the Bug

npm create cloudflare offers a default name, but pressing enter does not use it.
```
~/code/tmp ❯ npm create cloudflare@latest

using create-cloudflare version 2.0.10

╭ Create an application with Cloudflare Step 1 of 3
│
╰ Where do you want to create your application? also used as application name
--steep-math-c101
Directory `` already exists and is not empty. Please choose a new name.
npm ERR! code 1
npm ERR! path /Users/maxime/Code/tmp
npm ERR! command failed
npm ERR! command sh -c create-cloudflare
```

See: https://asciinema.org/a/dUIWlyLITxgjFwRZ1rqHDYMMT

**Comment by irvinebroque** [comment]
I can reproduce this, but only **_after_** I first run `npm create cloudflare@latest` once successfully. It seems like some kind of state for which directory to used is getting persisted across executions, or reset in an odd way?

**Comment by petebacondarwin** [comment]
This was fixed in this PR: https://github.com/cloudflare/workers-sdk/pull/3465 and will be included in 2.0.11
